### PR TITLE
Level0 hello

### PR DESCRIPTION
**Pull request by MaximKalinin**

### CODE REVIEW

File: src/index.js
Comment (lurenzsgp) [inline src/index.js:3]: Use `let` instead

File: src/index.js
Comment (lurenzsgp) [inline src/index.js:5]: 🙈 indentation

File: src/index.js
Comment (lurenzsgp) [inline src/index.js:8]: 🙈 new line before

**Review by lurenzsgp — CHANGES_REQUESTED** [review]
Please use 4 spaces of indentation.

File: src/index.js
Comment (lurenzsgp) [inline src/index.js:4]: 🙈 empty line before

I forgot it before the logics block we like an empty line to facilitate the code reading

File: src/index.js
Comment (lurenzsgp) [inline src/index.js:7]: 🙈 Remove bracket for one-line blocks

File: src/index.js
Comment (Pigotz) [inline src/index.js:8]: This is fine!

I'll approve this PR but I also suggest you to give a read to [this](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals) article :+1:

**Review by Pigotz — APPROVED** [review]
:+1: 

After the reviews, don't merge the PR, just close it

**Review by lurenzsgp — APPROVED** [review]

**Pull request closed without merging**
